### PR TITLE
Filter content-sources repos by origin and content_type

### PR DESCRIPTION
**Pull request by jlsherrill**
These params will be added to content-sources soon.  And in the short future there will be other origin's available (Red Hat for example).  In the far future there will be other content_types (most likely). 

I haven't tested this change so tread lightly :)

Content sources PR: https://github.com/content-services/content-sources-backend/pull/383

But this shouldn't cause any problem going in earlier.

### CODE REVIEW

**Review by croissanne — APPROVED** [review]

**Comment by jlsherrill** [comment]
oh i see its generated, working on it

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ foobarApi.ts
 7. run api generation
 
 ```
-rpn run api
+npm run api
```
Comment (jlsherrill) [inline README.md:150]: i assume this was a typo?

**Comment by lucasgarfield** [comment]
Glad you found the instructions! We want to make this fully automated with `npm run api` (and we also want to catch changes to the APIs we consume via a Github action) but we aren't quite there yet...

**Comment by lucasgarfield** [comment]
We will also need to grep for every time we call `useListRepositoriesQuery` and add the new params there, too.

**Comment by lucasgarfield** [comment]
Thank you!!!

**Review by lucasgarfield — APPROVED** [review]

**Pull request merged**